components/layout/Footer.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import styled from "styled-components";
+import ActiveLink from "./ActiveLink";
+
+export function Footer() {
+  return (
+    <StyledFooter >
+      <ActiveLink href="/specials" activeClassName="active">
+        <a>Specials</a>
+      </ActiveLink>
+      <ActiveLink href="/shop" activeClassName="active">
+        <a>Shop</a>
+      </ActiveLink>
+      <ActiveLink href="/favourites" activeClassName="active">
+        <a>Favourites</a>
+      </ActiveLink>
+      <ActiveLink href="/checkout" activeClassName="active">
+        <a>Checkout</a>
+      </ActiveLink>
+    </StyledFooter>
+  );
+}
+
+const StyledFooter = styled.ul`
```

Comment:
Why is this an `ul`? Maybe `footer` might be more appropriate

Reply:
applied in final commit